### PR TITLE
SYS-56 | Cobra Cli integration

### PR DESCRIPTION
**Pull request by Denes-cilwal**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,202 @@
+
+                                 Apache License
+                           Version 2.0, January 2004
+                        http://www.apache.org/licenses/
+
+   TERMS AND CONDITIONS FOR USE, REPRODUCTION, AND DISTRIBUTION
+
+   1. Definitions.
```
Comment (dipeshdulal) [inline LICENSE:8]: @Denes-cilwal Why was the license added?

File: main.go [hunk 2]
```diff
@@ -13,4 +12,5 @@ func main() {
 
 	logger := lib.GetLogger()
 	fx.New(bootstrap.Module, fx.Logger(logger.GetFxLogger())).Run()
+	//cmd.Execute()
```
Comment (dipeshdulal) [inline main.go]: Please remove comment.

File: bootstrap/bootstrap.go [hunk 3]
```diff
@@ -72,8 +72,8 @@ func bootstrap(
 			logger.Info("-------------------------------------")
 
 			logger.Info("Migrating database schemas")
-			migrations.Migrate()
-
+			//migrations.Migrate()
+			go cobracliApp.Execute()
```
Comment (dipeshdulal) [inline bootstrap/bootstrap.go]: @Denes-cilwal Please remove comments.

**Review by dipeshdulal — CHANGES_REQUESTED** [review]
I have not looked by running the code. Will do tomrrow. 🙏 

I have added my partial review. 🙇

File: LICENSE (hunk 1 above)
Comment (Denes-cilwal) [reply LICENSE:8]: @dipeshdulal  Sir, this was added when we try to use cobra-generator, it adds boilerplate with it, Later it was replaced by the cobra library(manually).

**Comment by dipeshdulal** [comment]
Updated the CLI implementation. You can check the `random` command to know how to use the CLI command interfaces. 

Running command can be done by:
```sh
docker-compose exec web sh 
./__debug_bin random -n 20
```

Adding the new command to the CLI is done by adding a struct that implements the `cmd.Command` interface to the `RootCommand.commands` list. 

Check the random command for implementation details.

**Pull request merged**